nipoppy/workflows/dataset_init.py

```diff
@@ -92,6 +119,83 @@ def run_main(self):
             " to match your dataset"
         )
 
+    def _write_readmes(self) -> None:
+        if self.dry_run:
+            return None
+        for dpath, description in self.layout.dpath_descriptions:
+            fpath_readme = dpath / self.fname_readme
+            if description is None:
+                continue
+            if dpath.stem != "bids" or self.bids_source is None:
+                fpath_readme.write_text(f"{description}\n")
+            elif self.bids_source is not None and not fpath_readme.exists():
+                gh_org = "bids-standard"
+                gh_repo = "bids-starter-kit"
+                commit = "f2328c58238bdf2088bc587b0eb4198131d8ffe2"
+                path = "templates/README.MD"
+                url = (
+                    "https://raw.githubusercontent.com/"
+                    f"{gh_org}/{gh_repo}/{commit}/{path}"
+                )
+                response = requests.get(url)
+                fpath_readme.write_text(response.content.decode("utf-8"))
```

Comment:
the bids starter repo will be integrated into the bids website so that template file us likely to move so I preferred to pin a specific commit to avoid problems.